pkgs/applications/video/mkvtoolnix/default.nix

```diff
@@ -4,6 +4,7 @@
 , withGUI ? true
   , qtbase ? null
   , qtmultimedia ? null
+  , wrapQtAppsHook ? null
```

Comment:
even though it's supposed to be denote it's optional, weird to see extra whitespace.

Personal preference. Don't feel strongly about it.
```suggestion
, wrapQtAppsHook ? null
```

Reply:
I’ve just followed the style of the two lines above.

Reply:
either way, I don't have much of a preference. Just a slight preference :)